README.md

```diff
@@ -249,7 +184,7 @@ authenticator.use(
       // createTOTP: async (data) => {},
       // ...
     },
-    async ({ email, code, magicLink, form, request }) => {
+    async ({ email }) => {
       // You can determine whether the user is authenticating
       // via OTP code submission or Magic-Link URL and run your own logic.
       if (form) console.log('Optional form submission logic.')
```

Comment:
If `form`/`magicLink` is no longer a property, how does the user know if the code submission was via a magic link or form? As far as I know, we are only passing `email`, right? Does the email include some hints?

It's not a big issue, though, but it could be useful for folks to call an API based on the user's click or method used to authenticate (code submission or magic-link).

I could be completely wrong about this, just asking for your thoughts.

Reply:
The `email` alone does not give any indication whether it was authenticated via OTP code or Magic-Link URL. I'm struggling to come up with a use case where knowing how the `email` was authenticated affects the what `user` is actually returned from `verify()`. 

`remix-auth-totp` seems not really designed for observability to, say, track OTP code submissions vs Magic-Link URL. If observability were to become a requirement for `remix-auth-totp`, would prefer we took the time to think through and design that out.

My preference would be to leave as is and see if users ask for more. And then we can really explore those use cases with them. However, I'm not prepared to die on that hill and if your gut says we should add additional properties, just let me know and I'll add them.